src/mranderson/move.clj

```diff
@@ -253,8 +252,9 @@
                                          (find-and-replace-platform-specific-subforms opposite-platform ns-loc))
                                     [[] ns-loc])
 
-        new-ns-form             (future (replace-in-ns-form ns-loc old-sym new-sym watermark))
-        new-source-sans-ns      (future (replace-in-source source-sans-ns old-sym new-sym))
+        ;; we use `delay` because the computations may not be needed, depending on the subsequent conditionals:
```

Comment:
hm... this may not be entirely true as both `new-ns-form` and `new-source-sans-ns` will be derefed in all branches of the subsequent conditionals. what am i missing?